src/main/scala/com/lucidworks/spark/util/SolrSupport.scala

```diff
@@ -449,19 +449,19 @@ object SolrSupport extends Logging {
     val zkStateReader: ZkStateReader = solrClient.getZkStateReader
     val clusterState: ClusterState = zkStateReader.getClusterState
     var collections = Array.empty[String]
-
-    if (clusterState.hasCollection(collection)) {
-      collections = Array(collection)
+  for(col <- collection.split(",")) {
```

Comment:
The indentation is still not right. Make sure the indentation aligns with the code above